src/microbe_stage/Microbe.cs

```diff
@@ -1058,13 +1058,15 @@ public void ApplyPropertiesFromSave(Microbe microbe)
     internal void SuccessfulScavenge()
     {
         GameWorld.AlterSpeciesPopulation(Species,
-            Constants.CREATURE_SCAVENGE_POPULATION_GAIN, "successful scavenge");
+            Constants.CREATURE_SCAVENGE_POPULATION_GAIN,
+            TranslationServer.Translate("MICROBE_SUCC_SCAV"));
```

Comment:
I don't think the ID strings should be abbreviated to make it easier to read them raw in the source code.
Also I'm not sure if the translation should happen here, I feel like the translation on these actions should happen when printing out this info, not during swimming around when this code is ran.